### PR TITLE
Torus constructor

### PR DESCRIPTION
**Pull request by stla**

### CODE REVIEW

File: src/primitives/torus.jl [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  major::T
+  minor::T
+end
+
+Torus(center::Point{3,T}, normal::Vec{3,T}, major, minor) where {T} = 
+  Torus(center, normal, T(major), T(minor))
+
+function Torus(center::Tuple, normal::Tuple, major, minor)
+  Torus(Point(center), Vec(normal), major, minor)
+end
```
Comment (juliohm) [inline src/primitives/torus.jl]:
```suggestion
Torus(center::Tuple, normal::Tuple, major, minor) =
  Torus(Point(center), Vec(normal), major, minor)
```

File: src/sampling/regular.jl [hunk 2]
```diff
@@ -172,4 +172,4 @@ end
 function sample(rng::AbstractRNG, grid::CartesianGrid,
                 method::RegularSampling)
   sample(rng, boundingbox(grid), method)
-end
\ No newline at end of file
+end
```
Comment (juliohm) [inline src/sampling/regular.jl]: please remove this EOF change

File: test/primitives.jl [hunk 3]
```diff
@@ -431,4 +431,10 @@
     @test !isconvex(s)
     @test isnothing(boundary(s))
   end
+
+  @testset "Torus" begin
+    t = Torus(T.((1,1,1)), T.((1,0,0)), 2, 1)
+    @test P3(1,1,-1) ∈ t
+    @test P3(1,1,1) ∉ t
+  end
```
Comment (juliohm) [inline test/primitives.jl]: Need tests for the accessor functions

**Review by juliohm — CHANGES_REQUESTED** [review]
Review attached.

**Comment by juliohm** [comment]
@stla did you have a chance to review these suggestions?

File: test/primitives.jl (hunk 3 above)
Comment (stla) [reply test/primitives.jl]: What are the accessor functions? The radii, the center, etc?

Comment (juliohm) [reply test/primitives.jl]: Exactly, the functions that return properties of the concept.

**Review by juliohm — APPROVED** [review]

**Comment by juliohm** [comment]
Thank you @stla for another useful addition 👏🏽 Should we start a new PR with the sampling method you already have implemented? You can add the method and some tests so that we can start visualizing the Torus effortlessly.

**Pull request merged**